task_netcore/task4/Client/src/modules/AccountForm/views/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Paper, TextField, Button, FormControl, FormHelperText, withStyles } from '@material-ui/core';
+import { Field, reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
+import styles from './styles';
+import { applicationRoutes } from '../../../Constants';
+
+const renderField = ({ input, label, type, meta: { touched, error }, margin }) => (
+	<div>
```

Comment:
Try to write the whole project in one code style. Check tabulation everywhere in the project. Some files have x2 tabulation and some have x4.